xwiki-pro-macros-ui/src/main/resources/Confluence/Macros/Expand.xml

```diff
@@ -0,0 +1,566 @@
+        <classType>com.xpn.xwiki.objects.classes.BooleanClass</classType>
+      </supportsInlineMode>
+      <visibility>
+        <cache>0</cache>
+        <disabled>0</disabled>
+        <displayType>select</displayType>
+        <freeText>forbidden</freeText>
+        <largeStorage>0</largeStorage>
+        <multiSelect>0</multiSelect>
+        <name>visibility</name>
+        <number>6</number>
+        <prettyName>Macro visibility</prettyName>
+        <relationalStorage>0</relationalStorage>
+        <separator>|</separator>
+        <separators>|</separators>
+        <size>1</size>
+        <unmodifiable>0</unmodifiable>
+        <values>Current User|Current Wiki|Global</values>
+        <classType>com.xpn.xwiki.objects.classes.StaticListClass</classType>
+      </visibility>
+    </class>
+    <property>
+      <async_cached>0</async_cached>
+    </property>
+    <property>
+      <async_context/>
+    </property>
+    <property>
+      <async_enabled>0</async_enabled>
+    </property>
+    <property>
+      <code>{{velocity}}
+$xwiki.ssx.use('Confluence.Macros.Expand')
+#if (!$expandMacroColllapseId)
+  #set ($expandMacroColllapseId = 0)
+#else
+  #set ($expandMacroColllapseId = $expandMacroColllapseId + 1)
+#end
+#set ($opened = $xcontext.action == 'edit')
+#set ($accorionId = "accordion-$expandMacroColllapseId")
+#set ($toggleId = "toggle-$expandMacroColllapseId")
+#set ($expandId = "collapse-$expandMacroColllapseId")
+
+{{html clean="false"}}
+&lt;div class="panel-group confluence-expand-macro" id="${accorionId}" role="tablist"&gt;
+  &lt;div class="panel panel-default"&gt;
+    &lt;div class="panel-heading" role="tab" id="${toggleId}"&gt;
+      &lt;h4 class="panel-title"&gt;
+        &lt;a role="button" data-toggle="collapse" data-parent="#${accorionId}" href="#${expandId}" #if($opened)aria-expanded="true"#end aria-controls="${expandId}"&gt;#*
+          *#$!wikimacro.parameters.title#*
+        *#&lt;/a&gt;
+      &lt;/h4&gt;
+    &lt;/div&gt;
+    &lt;div id="${expandId}" class="panel-collapse collapse #if($opened)in#end" role="tabpanel" aria-labelledby="${toggleId}"&gt;
+      &lt;div class="panel-body"&gt;
+        {{/html}}
+
+          {{wikimacrocontent /}}
+
+        {{html clean="false"}}
+      &lt;/div&gt;
+    &lt;/div&gt;
+  &lt;/div&gt;
+ &lt;/div&gt;
+{{/html}}
+
+{{/velocity}}</code>
+    </property>
+    <property>
+      <contentDescription/>
+    </property>
+    <property>
+      <contentJavaType>Wiki</contentJavaType>
+    </property>
+    <property>
+      <contentType>Optional</contentType>
+    </property>
+    <property>
+      <defaultCategory/>
+    </property>
+    <property>
+      <description>Add the Expand macro to your page to provide content in an expandable / collapsible section. 
+
+This is one of Confluence's most popular macros. It's great for:
+
+    visually reducing the amount of information on a page
+    breaking process information down into clickable steps
```

Comment:
```suggestion
    visually reducing the amount of information on a page,
    breaking process information down into clickable steps,
```

Adding the comma ensures that the text is property read when the white space is stripped.